File: examples/evchargers/referenceData/node/getReferenceData.js
const { EvchargersService } = require("m3o/evchargers");

// Retrieve reference data as used by this API and in conjunction with the Search endpoint
async function getReferenceData() {
  let evchargersService = new EvchargersService(process.env.MICRO_API_TOKEN);
  let rsp = await evchargersService.referenceData({});
  console.log(rsp);
}

getReferenceData();

File: examples/evchargers/referenceData/go/getReferenceData.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/evchargers"
)

// Retrieve reference data as used by this API and in conjunction with the Search endpoint
func GetReferenceData() {
	evchargersService := evchargers.NewEvchargersService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := evchargersService.ReferenceData(&evchargers.ReferenceDataRequest{})
	fmt.Println(rsp, err)
}
